modules/installation-and-upgrade/pages/install-vm.adoc

```diff
@@ -21,7 +21,7 @@ These are suitable for a quick test installation, such as a server with one clie
 If you want to use a production environment, review the requirements listed in xref:hardware-requirements.adoc[].
 ====
 
-In the following table replace {sles-version}-{sp-version} with the actual product version string.
+In the following table replace {sles-version}-{sp-base} with the actual product version string.
```

Comment:
```suggestion
In the following table replace {sles-version} {sp-base} with the actual product version string.
```

Reply:
I think the best would be if we'd tag this with `[literal]` here.  But that would be something for think about later. Applying your suggesting now